dev/src/dev.clj

```diff
@@ -1,5 +1,20 @@
 (ns dev
-  (:require [metabase db handler plugins server]))
+  "Put everything needed for REPL development within easy reach"
+  (:require [metabase
+             [core :as mbc]
+             [db :as mdb]
+             [handler :as handler]
+             [plugins :as pluguns]
+             [server :as server]
+             [util :as u]]
+            [metabase.api.common :as api-common]
+            [metabase.models.interface :as mi]
+            [toucan.db :as db]))
```

Comment:
the linters are failing because `toucan.db` isn't used in the namespace

from Eastwood:

```
dev/src/dev.clj:1:1: unused-namespaces: Namespace toucan.db is never used in dev
```